db/migrate/20250725050517_create_ticket_log_changes.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CreateTicketLogChanges < ActiveRecord::Migration[7.2]
+  def change
+    reversible do |dir|
+      dir.up do
+        create_table :ticket_log_changes do |t|
+          t.references :ticket_log, null: false
+          t.string :field_name, null: false
+          t.string :field_value, null: false
+          t.timestamps
+        end
```

Comment:
`create_table` is inherently reversible and does not need to go into a `reversible` block.

Reply:
Okay changed.